public/app/plugins/panel/table/TablePanel.tsx

```diff
@@ -79,7 +79,7 @@ export function TablePanel(props: Props) {
 
   const names = frames.map((frame, index) => {
     return {
-      label: getFrameDisplayName(frame),
+      label: `${frame.refId}: ${getFrameDisplayName(frame)}`,
```

Comment:
hmm, i think we just need to pass along the `index`? i see there is already `refId` handling in there.

https://github.com/grafana/grafana/blob/a3ada255795d4e54f5b05bc5bb33aab1ac69c735/packages/grafana-data/src/field/fieldState.ts#L12-L52

Reply:
The refId use in that function will only show if there are no field names though, right? What if we took that logic out, and just had it be "Series" So it would be `A: Series`, `B: Series` etc.